frontend/src/modules/Core/Constants/Theme.ts

```diff
@@ -1,8 +1,13 @@
 export const colors = {
+  mediumviolet: '#C794EE',
+  verylightviolet: '#FCFAFF',
   black: '#222222',
   white: '#FFFFFF',
   darkpurple: '#CE9EF2',
   lightviolet: '#E5CEFA',
   red: '#FF6584',
   yellow: '#FFFFB1',
+  verylightgreen: '#F0F9F4',
```

Comment:
we should probably clean up the color nomenclature more idiomatically eventually